blog/articles/get-started-knative-eventing.md

```diff
@@ -0,0 +1,93 @@
+| Title  | LinkTitle | Date | Description | Type | Aliases
+| ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | 
+| Getting started with Knative Eventing using Bitcoin transaction data  | Get Started with Knative Eventing  | 2020-05-01  | Guided walkthrough on using Knative Eventing  | Blog  | /blog/2010-05-01-getting-started-knative-evetning  |
```

Comment:
I think this needs to be in [Hugo frontmatter format](https://gohugo.io/content-management/front-matter/#front-matter-formats). I'd suggest using the `yaml` format, like so:

```suggestion
---
title: Getting started with Knaitve Eventing using Bitcoin transaction data
linkTitle: Bitcoin transaction data and Eventing
date: 2020-05-01
description: Walkthrough of implementing a Bitcoin transaction data source for Knative Eventing
type: "blog"
---
```